test/Interop/Cxx/ergonomics/implicit-computed-properties-module-interface.swift

```diff
@@ -265,3 +265,20 @@
 // CHECK-NEXT:    func getValue() -> Int32
 // CHECK-NEXT:    mutating func setValue(_ i: Int32)
 // CHECK-NEXT: }
+
+
+// CHECK:      struct SnakeCaseGetter {
+// CHECK-NEXT:    init()
+// CHECK-NEXT:    init(value: Int32)
+// CHECK-NEXT:    var value = 42;
+// CHECK-NEXT:    func get_foo() -> Int32
+// CHECK-NEXT:    mutating func set_foo(_ i: Int32)
+// CHECK-NEXT: }
```

Comment:
Notice that the computed property `foo` isn't here. I think that's because of this check:
```
    if (nameKind != NameKind::title && nameKind != NameKind::camel &&
        nameKind != NameKind::lower)
      return Kind::unkown;
```

On line 25. You need to update this to tell the compiler we can import snake case getters setters now :)

Reply:
oh yep, i missed that 😂